x-pack/plugins/ml/public/application/explorer/alerts/alerts_panel.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonGroup, EuiNotificationBadge, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { AlertConsumers } from '@kbn/rule-data-utils';
+import React, { useState, type FC } from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import { ML_ALERTS_CONFIG_ID } from '../../../alerting/anomaly_detection_alerts_table/register_alerts_table_configuration';
+import { CollapsiblePanel } from '../../components/collapsible_panel';
+import { useMlKibana } from '../../contexts/kibana';
+import { useAnomalyExplorerContext } from '../anomaly_explorer_context';
+import { AlertsSummary } from './alerts_summary';
+import { AnomalyDetectionAlertsOverviewChart } from './chart';
+import { statusNameMap } from './const';
+
+export const AlertsPanel: FC = () => {
+  const {
+    services: { triggersActionsUi },
+  } = useMlKibana();
+
+  const [isOpen, setIsOpen] = useState(true);
+  const [toggleSelected, setToggleSelected] = useState(`alertsSummary`);
+
+  const { anomalyDetectionAlertsStateService } = useAnomalyExplorerContext();
+
+  const countByStatus = useObservable(anomalyDetectionAlertsStateService.countByStatus$);
+  const alertsQuery = useObservable(anomalyDetectionAlertsStateService.alertsQuery$, {});
+
+  const alertStateProps = {
+    alertsTableConfigurationRegistry: triggersActionsUi!.alertsTableConfigurationRegistry,
+    configurationId: ML_ALERTS_CONFIG_ID,
+    id: `ml-details-alerts`,
+    featureIds: [AlertConsumers.ML],
+    query: alertsQuery,
+    showExpandToDetails: true,
+    showAlertStatusWithFlapping: true,
+  };
+  const alertsStateTable = triggersActionsUi!.getAlertsStateTable(alertStateProps);
+
+  const toggleButtons = [
+    {
+      id: `alertsSummary`,
+      label: i18n.translate('xpack.ml.explorer.alertsPanel.summaryLabel', {
+        defaultMessage: 'Summary',
+      }),
+    },
+    {
+      id: `alertsTable`,
+      label: 'Details',
+    },
+  ];
+
+  return (
+    <>
+      <CollapsiblePanel
```

Comment:
Is it possible to reduce the padding for this (and the annotations) panel? Could save some vertical height.

<img width="169" alt="image" src="https://github.com/elastic/kibana/assets/7405507/1c9609de-15cb-4066-be11-a8f3e0c7bc14">

Reply:
I'd check this with @mdefazio, wdyt?

Reply:
Spoke with Dima about this and we think it makes sense to adopt a similar style for many of the collapsible panels.  So once he adopts this more broadly across ML (sounded like his next step), we can take a look and see what it means across usages for smaller padding.  Not against it, but this feels outside of this PR and would like to focus on this specifically